### DISCUSSION TITLE
Untranslated text in callhusks event

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14854] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Untranslated text in callhusks event

![image](https://github.com/user-attachments/assets/33ea1001-3183-4419-87c8-8cfabc5a6e05)

<EventText.callhusks.c1>I could call more husks if needed.</EventText.callhusks.c1>
<EventText.callhusks.c2>I think there's still some power left in the husk caller, I could probably use it one more time if needed…</EventText.callhusks.c2>

![image](https://github.com/user-attachments/assets/93ea990d-ca97-4e50-98ea-f61219240eb8)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/e9fcb17e1a0333d7271b8730e68b9e9e7f952362. The fix unfortunately didn't make it for the Unto the Breach update (v1.16), but should be coming up in the next one.
